src/fsharp/FSharp.Core/prim-types.fs

```diff
@@ -637,6 +637,19 @@ namespace System
 #else   
 #endif
 
+namespace Microsoft.FSharp.Core
+
+    open System
+    open Microsoft.FSharp.Core
+
+    module Resources =
```

Comment:
it cannot, like the old ErrorStrings, this doesn't work for inline function
An example [Array.maxBy](https://github.com/Microsoft/visualfsharp/blob/275b832e9dd1a4bd64ed3accd218384b901be1d2/src/fsharp/FSharp.Core/seq.fs#L1647) is inline, so when inlined the code need to access the error string
it cannot use internal functions, like `SR.GetString` , because the inlined code cannot access it